### PR TITLE
Ghg equivalencies

### PR DESCRIPTION
**Pull request by BradHN1**

### CODE REVIEW

**Comment by frimpongopoku** [comment]
We need to be careful here. I cant sign in here, and its not because I am not registered. Its says, `resource_not_found` I havent looked more into it. Its probably my local env. Just wanted to have a feel of what the GHG eq looked like on the admin side.

File: v2.0/app/containers/MassEnergizeSuperAdmin/_FormGenerator/icon_files.json [hunk 1]
```diff
@@ -0,0 +1,790 @@
+{
+    "ICON_FILES": [
```
Comment (frimpongopoku) [inline v2.0/app/containers/MassEnergizeSuperAdmin/_FormGenerator/icon_files.json:2]: Also what is happening here? I am actually curious. Is there a reason we need such a long list and such a large file? 
Are trying to implement fontawesome icons here...? 
Why are we not using the npm package instead?

**Comment by BradHN1** [comment]
The resource_not_found is probably from the API routes changing on Dev (no more ‘/v3/‘).

**Comment by BradHN1** [comment]
There may be better ways to do this, but we use fontawesome icons for the home page, so I thought it ok to use them for the GHG equivalencies as well.  It was requested that we allow a greater range of icons than the few that we had used as examples, so I downloaded a list and allow selecting it from a pull-down.  

Happy to do another way if you suggest a better.  Not sure what npm package you are referring to.

**Review by Opoku-Agyemang — APPROVED** [review]

**Pull request merged**